dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/ui/UISoftwareResource.java

```diff
@@ -67,39 +68,63 @@
   @GET
   @Produces(APPLICATION_JSON)
   public List<MavenArtifactVersion> getArtifacts() {
-    val results = storageClientService.getVersions();
+    val results = softwareService.getMavenVersions();
+    return results;
+  }
+
+  @Path("/icgc-get/versions")
+  @GET
+  @Produces(APPLICATION_JSON)
+  public List<ArtifactFolder> getICGCGetArtifacts() {
```

Comment:
Lets be consistent with the casing on `ICGC`.  Seems like most are `Icgc`. That's fine, but change the others like this one. Thanks.